### PR TITLE
Resolved issue #453

### PR DESCRIPTION
**Pull request by drpayyne**
### Description

- Added MFTF test as required by issue
- Changed css `position` of the sign in button since the given `z-index` doesn't affect statically positioned elements

### Fixed Issues
- magento/adobe-stock-integration#453: [MFTF] User sees credits available on their account

### Manual testing scenarios
Make sure the ENV file has the following variables
-  `ADOBE_STOCK_USER_EMAIL=<adobe_stock_email_id>`
-  `ADOBE_STOCK_USER_PASSWORD=<adobe_stock_password>`

```bash
$ vendor/bin/mftf run:test AdminAdobeStockSignedInCreditsVisibleTest --remove
```

### CODE REVIEW

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSignedInCreditsVisibleTest.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <after>
+            <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
+            <actionGroup ref="logout" stepKey="logout"/>
+        </after>
+        <actionGroup ref="AdminAdobeStockClickSignInActionGroup" stepKey="clickOnSignIn"/>
+        <actionGroup ref="AdminAdobeStockImsPopupSignInFillUserDataActionGroup" stepKey="fillUserCredentials"/>
+        <actionGroup ref="AdminAdobeStockImsPopupClickSignInActionGroup" stepKey="clickSignInImsPopup"/>
+        <actionGroup ref="AdminAdobeStockAssertUserLoggedActionGroup" stepKey="assertUserLoggedIn"/>
+
+        <click selector="{{AdobeStockSection.userNameButton}}" stepKey="openAccountMenu"/>
```
Comment (Nazar65) [inline AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSignedInCreditsVisibleTest.xml]: I think we can move assertion to action group, also click on menu and others,

Comment (drpayyne) [reply AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSignedInCreditsVisibleTest.xml]: Thank you, @Nazar65. I've fixed my code according to your suggestion.

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSignedInViewCreditsActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAdobeStockSignedInViewCreditsActionGroup">
+        <click selector="{{AdobeStockSection.userNameButton}}" stepKey="openAccountMenu"/>
+        <see userInput="AVAILABLE" selector="{{AdobeStockSection.userQuota}}" stepKey="seeAvailableText"/>
```
Comment (eduard13) [inline AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSignedInViewCreditsActionGroup.xml]: Could you please move and store the following texts to a data entity?

**Review by eduard13 — DISMISSED** [review]

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSignedInViewCreditsActionGroup.xml (hunk 2 above)
Comment (drpayyne) [reply AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSignedInViewCreditsActionGroup.xml]: @eduard13 Done, thanks!

**Review by filmaj — APPROVED** [review]
LGTM! What does the CSS change do?

**Comment by drpayyne** [comment]
@filmaj The sign in button already had a `z-index` property but turns out that, this property **doesn't work** on statically positioned elements. So I changed the `position` property from `initial` to `relative` to enable to `z-index` which was already set. This `z-index` is required for the button to be on top of the profile popup container (which has no content, but overlaps on the signin button) for MFTF tests to simulate a click on the sign in button, else the signin button element is unreachable.

Thank you!

**Comment by filmaj** [comment]
Thank you @drpayyne !!!

**Pull request merged**